vessim/cosim/energy_system_interface.py

```diff
@@ -156,7 +156,7 @@ def init_get_routes(self, app: FastAPI) -> None:
             app: The FastAPI app to add the GET routes to.
         """
         # store attributes and its initial values in Redis key-value store
-        redis_init_content = {
+        redis_init_content: Dict[Union[str, bytes], Union[bytes, float, int, str]] = {
```

Comment:
what was the error here? Can we actually have `byte` keys here sometimes?

Reply:
I don't think we can expect bytes there, but the error was triggered in line 171: self.redis_docker.redis.mset(redis_init_content)
mset expects this specific type: "Mapping[Union[str, bytes], Union[bytes, float, int, str]]" but only "Dict[str, Any]" was provided, therefore specifying the structure of the dictionary like this solved the problem.